openapi-specs/schema.json

```diff
@@ -5458,6 +5479,68 @@
             }
           }
         }
+      },
+      "get": {
+        "summary": "Get a specific tenant",
+        "description": "get a specific tenant for the given class",
+        "operationId": "tenants.get.one",
+        "tags": [
+          "schema"
+        ],
+        "parameters": [
+          {
+            "name": "className",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "tenantName",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "consistency",
+            "in": "header",
+            "required": false,
+            "default": true,
+            "type": "boolean",
+            "description": "If consistency is true, the request will be proxied to the leader to ensure strong schema consistency"
+          }
```

Comment:
This could be a TenantParameter type that could be reused for input and output?

Reply:
Sounds reasonable 👍. @nathanwilk7 something you can consider in the follow up PR?. I'm merging this one to unblock

Reply:
Ah couldn't merge because of this CI failure. 
<img width="370" alt="Screenshot 2024-11-15 at 09 20 51" src="https://github.com/user-attachments/assets/6c986edc-d224-428c-8e82-9db53529174e">

may be @dirkkul feedback could reduce this.

Reply:
I learned. it won't be a blocker :)